tests/networkpolicy_test.go

```diff
@@ -176,11 +288,17 @@ func skipNetworkPolicyRunningOnKindInfra() {
 	}
 }
 
-func createVMICirros(virtClient kubecli.KubevirtClient, namespace string, labels map[string]string) *v1.VirtualMachineInstance {
+//TODO: Change this when [1] is merged
+//      [1] https://github.com/kubevirt/kubevirt/pull/4035/commits/886fd42d7b0e80f42c2b57df747850599637cc1b
```

Comment:
Link a PR, not a commit. Once we forcepush to the PR, the commit will be gone

Reply:
Done